extra/1-currency-conversion.js

```diff
@@ -15,7 +17,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(pounds) {
+  let NoFee = pounds - (pounds / 100)
```

Comment:
this is actually yesFee :) 
ok, here is the advice: 
- always name variables with first small letter, so it should be either noFee or in your case it's actually `transferableAmount` 
- the formula would be easier to read if you write it his way: 
```js
let transferableAmount = pound (1 - 0.01);
```
this give the same result, but it explicitly shows that you subtract 1% from the full amount.